.github/workflows/pages.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release docs
+
+on:
+  push:
+    branches:
+      - master
+      - 'branch-**'
+    paths:
+      - 'docs/**'
```

Comment:
shouldn't the path be docs/source/something?

Reply:
no, we want to update GH Pages when docs are touched, not only content, but scripts too.